src/Sphere.cc

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  *
 */
+#include <gz/math/Inertial.hh>
```

Comment:
```suggestion
#include <optional>
#include <gz/math/Inertial.hh>
```

Reply:
Added in [1f16e1f](https://github.com/gazebosim/sdformat/pull/1304/commits/1f16e1f0a5454acc5a30a4b86b8f273b1535eb95)